documentation/Database-Schema-Review.md

```diff
@@ -0,0 +1,99 @@
+# Database Schema Review
+
+![Database Schema](images/database-schema-review.png)
+
+## Database Area description: Review
+
+### Definition
+
+The review is part of the process of the application in the workflow to be approved or rejected in the system. For example a Drug Registration application, after being submited, goes through stages. In each one of these stages at least one review is required. To have an application reviewed, the system counts with special users called Reviewers (or evaluators).
+
+The Reviewer may or may not have permission to submit the LOQ (List of Questions) back to the Applicant that will be allowed to edit this selected LOQ.
+
+Actions will define which permission is required to submit the reviewed application. Which means that for some applications on a stage it can be required **more than one** review to done for the same section.
+
+First done by Reviewers with lowel `review level` permission, to add comments and decisions. This reviews will sit there until another review, called **consolidation**, is done (on top of the first ones). And the it will be submitted back to the Applicant. In some cases even one more review can be done on top of the second one - more details about that coming soon.
+
+## Tables
+
+### review
+
+A single review is created per pair of **application-stage**. Although some applications may not required any review (which will be defined by the actions for the particular template and stage on the trigger `onApplicationSubmitted`).
```

Comment:
Yes. That's what I'm thinking we would have for each **review**.  See my question on the description of this PR, I wasn't sure because of the field `comment` on this table.
Yes, the pair seems unneeded. Removing it now.